app-admin/fleet/metadata.xml

```diff
@@ -3,7 +3,7 @@
 <pkgmetadata>
   <!-- maintainer-needed -->
   <longdescription lang="en">
-	</longdescription>
+    </longdescription>
```

Comment:
Just cleaning up a very minor repoman-reported white space issue whilst the opportunity arises.